src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_acl.erl

```diff
@@ -270,18 +275,30 @@ safe_fetch_ids(_Type, _OrgId, []) ->
     % Save the cost of a query for an empty list -
     % fairly common for users and to a lesser extent groups.
     [];
-safe_fetch_ids(Type, OrgId, Names) ->
-    Records = oc_chef_authz_db:authz_records_by_name(Type, OrgId, Names),
-    FoundNames = lists:sort(names_from_records(Records)),
-    GivenNames = lists:sort(Names),
-    case GivenNames -- FoundNames of
+safe_fetch_ids(Type, OrgId, Names) when not is_list(OrgId) ->
```

Comment:
This guard is a little ambiguous since a string will drop through to the `OrgIds` path.  `when is_binary` would be safer.